3p/bodymovinanimation.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {loadScript} from './3p';
+import {parseJson} from '../src/json';
+
+/**
+ * Produces the AirBnB Bodymovin Player SDK object for the passed in callback.
+ * @param {!Window} global
+ * @param {function(!Object)} cb
+ */
+
+function getBodymovinAnimationSdk(global, cb) {
+  loadScript(global, 'https://cdnjs.cloudflare.com/ajax/libs/bodymovin/4.13.0/bodymovin.js', function() {
+    cb(global.bodymovin);
+  });
+}
+
+export function bodymovinanimation(global) {
+  const dataReceived = parseJson(global.name)['attributes']._context;
```

Comment:
@lannka 

WDYT the best approach here would be? The use case is injecting `loop` and `animationData` *from* the AMP context into the iframe. Writing and parsing `_context` seems like undefined behavior to me as it's not using [`AmpContext`](https://github.com/ampproject/amphtml/blob/master/3p/ampcontext.js).

It seems to me like we need to change the signature of `getIframe` so that arbitrary data fields can be inserted, not just data attrs.

Reply:
@aghassemi I will also wait on this before landing.

Reply:
I suggest merging the PR and handling this later given there doesn't seem to be a solution for it at the moment

Reply:
@alanorozco Thanks for bring it up to attention.
The data should be passed in from the function param:

```
export function bodymovinanimation(global, data) {
```

Does that not work?

Reply:
The `data` param only has the `data-*` attributes. None of the `<amp-bodymovin-animation>` components are data prefixed. 
So currently I am pushing these onto the opt_context argument here:
On the JS side here is the code https://github.com/ampproject/amphtml/pull/13810/files#diff-c8b48553d981913dfa40ed22dbb1a18bR74